reprozip/reprozip/tracer/trace.py

```diff
@@ -292,13 +296,17 @@ def tty_prompt(prompt, chars):
             ostream.flush()
 
 
-def trace(binary, argv, directory, append, verbosity=1):
+def trace(binary, argv, directory, append, verbosity='unset'):
     """Main function for the trace subcommand.
     """
+    if verbosity != 'unset':
+        warnings.warn("The 'verbosity' parameter for trace() is deprecated. "
+                      "Please set a level on the 'reprozip' logger intead.",
```

Comment:
s/intead/instead/

(invaluable feedback, I know :] )

Reply:
On the contrary, thanks for looking at this at all! Fixed by 9d6c5c21